### DISCUSSION TITLE
i cant delete repository.

### DISCUSSION THREAD

**Discussion by pupupu270** [community/community#138613] [Repositories] [answered]
### Select Topic Area

Question

### Body

im beginner at github.

so,want to create some repositories,and want to delete them.

but i cant do it.

delete steps is..

1,select repository
2,click setting
3,click delete repository in danger zone.

but stepp3 does not work.

i cant click it. push my mouse button. but not work.

html console error, i watched.
in console at github site, written below

Uncaught ReferenceError: HTMLDialogElement is not defined
    eh dialog_helper.js:22
    connectedCallback dialog_helper.js:84
    357 dialog_helper.js:129
    Webpack 9

how do i define HTML dialog element?

**Comment by izdannychu** [comment]
Can i get more info about it?

**Comment by Sin-cy** [comment]
Hmmm. Okay so I am gonna guess from the button not being able to click delete. 
You will basically have to type in the repo name it tells you to correctly first then only the button will be clickable. 
Like shown in the image below. If the problem is not because of this, then I think its the issues on github end. Maybe browser cache but that would be quite strange.
<img width="692" alt="Screenshot 2567-09-14 at 15 57 55" src="https://github.com/user-attachments/assets/de6ee983-fed7-427e-92ae-653b36acae8b">

**Reply by pupupu270** [reply]
thank you reply.

my screenshot is below.

i can open this site. but 'Delete this respositoy' button, i cant click and cant go next.

what info like would you need more?
![09142020](https://github.com/user-attachments/assets/6b14a647-33aa-4d70-8567-7d0158dc5fa3)

**Comment by jovanovic-djo** [comment, marked as answer]
@Sin-cy suggested good solution, but regarding that it might not resolve the issue, I suggest you to clear browser's cache and cookies, then do hard load (ctrl + F5) of Github page.
Another suggestion is to try deleting repository from other browsers, for example, if you are using Chrome, log in to your Github account with Opera, Firefox or Brave, and then try to delete repo.

I hope it works.

**Comment by pupupu270** [comment]
thankyou reply.

i try your advise.

first, "delete browser's cookie", it is not work well.

hard load isnot too.

next other browser, i use to delete repository.
i try to do it by mobile phone.
so,
 it do work!!

i CAN delete target repository.

just only PC browser, unfortunately it is not going well.

but for now, i can do what i want to.

thanks for your advice.

for reference,

using OS is linuxmint 20.1
using browser is firefox 90.0

if i got into trouble next, please give me your cooperate.

**Reply by jovanovic-djo** [reply]
I am glad that it helped.

**Comment by Ajit-pawara** [comment]
Hello, I would like to delete the forked repository GC.github.io from my account. Since I do not have admin rights, I cannot delete it manually. Please assist me in removing this fork. Thank you!

**Reply by pupupu270** [reply]
thankyou for your reply. but i have been solved by using another  brower.bye!
